staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -67,6 +83,8 @@ type AuditLogOptions struct {
 	MaxBackups int
 	MaxSize    int
 	Format     string
+	// Defaults to asynchronous batch events.
```

Comment:
Extract to a separate struct `AuditBufferedOptions` and add it as a field here instead.

The resulting class will look like this:

```go
type AuditLogOptions struct {
    AuditBufferedOptions
 	Path       string
 	MaxAge     int
  	MaxBackups int
  	MaxSize    int
  	Format     string
}
```

Reply:
Confused, the mode is not only used for buffered backend.

Reply:
I mean buffered and blocking, only have the mode difference in options.

Reply:
For now, it will grow

Reply:
Now we buffered backend do not need followingoptions, only the underlying log backend need them.
```
 	Path       string
 	MaxAge     int
  	MaxBackups int
  	MaxSize    int
  	Format     string
```